src/shared/components/mutationTable/MutationTable.tsx

```diff
@@ -0,0 +1,340 @@
+        };
+
+        this._columns[MutationTableColumnType.MUTATION_TYPE] = {
+            name: "Mutation Type",
+            render:MutationTypeColumnFormatter.renderFunction,
+            download:MutationTypeColumnFormatter.getTextValue,
+            sortBy:(d:Mutation[])=>MutationTypeColumnFormatter.getDisplayValue(d),
+            filter:(d:Mutation[], filterString:string, filterStringUpper:string) =>
+                MutationTypeColumnFormatter.getDisplayValue(d).toUpperCase().indexOf(filterStringUpper) > -1
+        };
+
+        this._columns[MutationTableColumnType.MUTATION_ASSESSOR] = {
+            name: "Mutation Assessor",
+            render:MutationAssessorColumnFormatter.renderFunction,
+            download:MutationAssessorColumnFormatter.getTextValue,
+            sortBy:(d:Mutation[])=>MutationAssessorColumnFormatter.getSortValue(d),
+            filter:(d:Mutation[], filterString:string, filterStringUpper:string) =>
+                MutationAssessorColumnFormatter.filterValue(d).toUpperCase().indexOf(filterStringUpper) > -1,
+            visible: false
+        };
+
+        this._columns[MutationTableColumnType.COSMIC] = {
+            name: "COSMIC",
+            render: (d:Mutation[])=>CosmicColumnFormatter.renderFunction(d, this.props.cosmicData),
+            sortBy: (d:Mutation[])=>CosmicColumnFormatter.getSortValue(d, this.props.cosmicData),
+            download: (d:Mutation[])=>CosmicColumnFormatter.getDownloadValue(d, this.props.cosmicData),
+            tooltip: (<span>COSMIC occurrences</span>),
+            defaultSortDirection: "desc"
+        };
+
+        this._columns[MutationTableColumnType.ANNOTATION] = {
+            name: "Annotation",
+            render: (d:Mutation[]) => (AnnotationColumnFormatter.renderFunction(d, {
+                hotspots: this.props.hotspots,
+                myCancerGenomeData: this.props.myCancerGenomeData,
+                oncoKbData: this.props.oncoKbData,
+                oncoKbEvidenceCache: this.props.oncoKbEvidenceCache,
+                pmidCache: this.props.pmidCache,
+                enableOncoKb: true,
+                enableMyCancerGenome: true,
+                enableHotspot: true
+            })),
+            sortBy:(d:Mutation[])=>{
+                return AnnotationColumnFormatter.sortValue(d,
+                    this.props.hotspots,
+                    this.props.myCancerGenomeData,
+                    this.props.oncoKbData);
+            }
+        };
+    }
+
+    @computed protected get orderedColumns(): MutationTableColumnType[] {
+        return _.sortBy(this.props.columns, (c:MutationTableColumnType)=>this._columns[c].order);
+    }
+
+    @computed protected get columns():Column<Mutation[]>[] {
+        return this.orderedColumns.reduce((columns:Column<Mutation[]>[], next:MutationTableColumnType) => {
+            columns.push(this._columns[next]);
+            return columns;
+        }, []);
+    }
+
+    public render()
```

Comment:
ideally they would not override render, they would just change whatever properties are needed to be specified. i.e. the columns and initialItemsPerPage, and anything else.